packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -100,16 +102,25 @@ const MobileDialog = (props: React.PropsWithChildren<TMobileDialog>) => {
                 <Div100vhContainer
                     className={classNames('dc-mobile-dialog__container', {
                         'dc-mobile-dialog__container--has-scroll': props.has_content_scroll,
-                        'dc-mobile-dialog__container--has-info-banner': info_banner,
+                        'dc-mobile-dialog__container--has-info-banner': info_banner || learn_more_banner,
                     })}
                     height_offset={props.content_height_offset || '8px'}
                 >
                     <ThemedScrollbars
-                        is_bypassed={!info_banner}
+                        is_bypassed={!info_banner && !learn_more_banner}
                         is_scrollbar_hidden
-                        className={info_banner ? classNames('dc-mobile-dialog__header-wrapper', header_classname) : ''}
+                        className={
+                            info_banner || learn_more_banner
+                                ? classNames('dc-mobile-dialog__header-wrapper', header_classname)
+                                : ''
+                        }
                     >
-                        <div className={classNames('dc-mobile-dialog__header', !info_banner && header_classname)}>
+                        <div
+                            className={classNames(
+                                'dc-mobile-dialog__header',
+                                (!info_banner || !learn_more_banner) && header_classname
```

Comment:
```suggestion
                                (!info_banner && !learn_more_banner) && header_classname
```
should `header_classname` be applied when there are no banners at all?